terraform/modules/pre-auth-lambda/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "name_prefix" {
+  type        = string
+  description = "(Required) Name prefix for resources created"
+}
+
+variable "common_tags" {
+  description = "common tags to apply to aws resources"
```

Comment:
you either need to add `(Required)` or add a default

Reply:
This is done.